openedx/features/edly/tests/test_middleware.py

```diff
@@ -70,6 +70,40 @@ def test_user_without_edly_organization_access(self):
                 target_status_code=200,
                 fetch_redirect_response=True
             )
+            user = auth.get_user(self.client)
+            assert not user.is_authenticated()
+
+            logger.check_present(
+                (
+                    LOGGER_NAME,
+                    'ERROR',
+                    'Edly user %s has no access for site %s.' % (self.user.email, self.request.site)
+                )
+            )
+
+    def test_user_without_edly_organization_access_and_with_frontend_logout_url(self):
+        """
+        Verify that logged in user gets redirected to logout page and valid log message response if user has no access with FRONTEND_LOGOUT_URL set.
+
+        Test that logged in user gets redirected to logout page and valid log message if user has no access for
+        request site's edly sub organization when FRONTEND_LOGOUT_URL is set.
+        """
+
+        with LogCapture(LOGGER_NAME) as logger:
+            expected_logout_url = '/logout'
+            setattr(settings, 'FRONTEND_LOGOUT_URL', expected_logout_url)
```

Comment:
Use `override_settings` decorator to set django settings values for a test
```python
@override_settings(GOOGLE_ANALYTICS_TRACKING_ID='UA-123456-1')
```

Reply:
Fancy 👍